release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -286,6 +286,75 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
+
+[id="ocp-4-10-storage-azure-file-csi-operator"]
+==== Persistent storage using the Microsoft Azure File CSI Driver Operator (Technology Preview)
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure File. The Azure File Driver Operator that manages this driver is in Technology Preview.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator].
+
+[id="ocp-4-10-storage-ibm-vpc-block-csi-operator"]
+==== Persistent storage using the IBM VPC Block CSI Driver Operator
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for IBM Virtual Private Cloud (VPC) Block. The IBM VPC Block Driver Operator that manages this driver is generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-ibm-vpc-block.adoc#persistent-storage-csi-ibm-vpc-block[IBM VPC Block CSI Driver Operator].
+
+[id="ocp-4-10-storage-vsphere-csi-operator"]
+==== Persistent storage using VMware vSphere CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for vSphere. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[vSphere CSI Driver Operator].
+
+[id="ocp-4-10-storage-azure-disk-csi-operator"]
+==== Persistent storage using Microsoft Azure Disk CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure Disk. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure.adoc#persistent-storage-csi-azure-disk[Azure Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-aws-efs-csi-operator"]
+==== Persistent storage using AWS Elastic File Storage CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Storage (EFS). This feature was previously introduced as a Technology Preview feature in {product-title} 4.9 and is now generally available and enabled by default in {product-title} 4.10.
+
```

Comment:
EFS csi operator is managed by OLM.

Reply:
I've removed the statement about being enabled by default.